modules/navier_stokes/test/tests/finite_volume/pins/channel-flow/porosity_jump/bernoulli-1d.i

```diff
@@ -0,0 +1,128 @@
+rho = 1.1
+advected_interp_method = 'upwind'
+velocity_interp_method = 'rc'
+
+[Mesh]
+  [mesh]
+    type = CartesianMeshGenerator
```

Comment:
Out of curiosity, is this treatment going to result in convergence issues if the interface between two porous media is in a detached vortex?

Reply:
I don't know